girder/api/describe.py

```diff
@@ -436,7 +436,7 @@ def __init__(self, templatePath=None):
         self.vars = {
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder - REST API Documentation',
+            'brandName': ModelImporter.model('setting').get(constants.SettingKey.BRAND_NAME),
```

Comment:
This is not getting updated until we restart the server. It probably needs an event handler too.